api/_utils/location-api.ts

```diff
@@ -7,7 +7,7 @@ type IPLocationData = {
 
 export const fetchCoordinates =
   async (ipAddress: string): Promise<[number, number]> => {
-    const response = await fetch(`https://ipapi.co/${ipAddress}/json/`);
+    const response = await fetch(`https://api.ipstack.com/${ipAddress}?access_key=${process.env.IPSTACK_API_KEY}`);
```

Comment:
The `https` endpoint is failing without a subscription. 😕

@samueldlay Let's discuss more out of band. I have an idea for trying multiple free-tier services with different quota policies.

Reply:
🤔👍